.github/workflows/ci.yml

```diff
@@ -41,12 +61,19 @@ jobs:
       - name: Build packages
         run: yarn lerna run prepack
       - run: ./bin/run whoami
-      - run: yarn lerna run test:acceptance
+        # on 'main' and 'release-' branches, these tests will be run as part of the acceptance tests.
+      - name: run smoke tests
+        if: github.ref_name != 'main' || !startsWith(github.head_ref, 'release-')
```

Comment:
There are two if statements, one to run the smoke tests and one to run the acceptance tests (they are opposites). I've done it this way for three reasons:
- I want to run only the smoke tests on PRs, not the full acceptance test suite
- I don't want to run the smoke tests separately for releases and merges to main (they will run as part of the full acceptance test suite)
- I want to keep both the smoke and acceptance tests in the "acceptance" job so that we can include it as part of the "needs" array in the "done" job.

Because of this, I don't think I want to move that if statement. If you think there is a better way to accomplish these things, though, I'd be open to hearing it.